AppCenter/AppCenterTests/MSChannelUnitDefaultTests.m

```diff
@@ -131,7 +131,6 @@ - (void)testLogsSentWithSuccess {
 
   // When
   dispatch_async(self.logsDispatchQueue, ^{
-
```

Comment:
This line break was removed by `clang-format`. I don't think we should re-add it, because if we do, this will be a discussion we have every time this file is modified.

Reply:
The same thing goes for all other line breaks that were removed before a comment but after a block. @ElektrojungeMS and I checked but there is no setting to disable this behavior in the format file.

Reply:
IMHO removing the empty line doesn't affect readability. clang-format's default behavior should be fine.

Reply:
My eyes is used to see a new line before a comment so it's a bit of a discomfort for me but sounds like we don't have a choice.